### PR TITLE
Siddiqui ph authors

### PR DESCRIPTION
**Pull request by nabsiddiqui**
I added my profile pic and my bio to the project team as requested by onboarding process: https://github.com/programminghistorian/jekyll/wiki/Onboarding-Process-for-New-Editors

### CODE REVIEW

**Comment by acrymble** [comment]
Something is not right @nabsiddiqui with the preview.

For some reason there is a second bio showing, but I'm not sure where you've put it because it doesn't look like it's in the files you've added to the pull request.

> Nabeel Siddiqui is a doctoral candidate in American Studies at the College of William and Mary, where he is also an inaugural fellow of the Digital Equality Lab. His research focuses broadly on the relationship between the humanities and computation, with a particular expertise on the cultural history of media, new media studies, digital humanities, information studies, and the history of science and technology. Currently, he is completing his dissertation entitled "Byting Out the Public," which looks at the historical relationship between the personal computer and the private sphere. 

Can you remove that from wherever it is?

**Comment by walshbr** [comment]
You have two bios in the authors data file - one from when you were an author and still a student.

**Comment by acrymble** [comment]
Can we get confirmation that the translations are ok? @programminghistorian/french-team @programminghistorian/spanish-team @programminghistorian/portuguese-team 

  bio:
    en: |
      Nabeel Siddiqui is an Assistant Professor of Digital Media at Susquehanna University.
    es: |
      Nabeel Siddiqui es profesor asistente de medios digitales en la Universidad de Susquehanna.
    fr: |
      Nabeel Siddiqui est professeur adjoint de médias numériques à l'Université Susquehanna.
    pt: |
      Nabeel Siddiqui é professor assistente de mídia digital na Susquehanna University.

**Comment by DanielAlvesLABDH** [comment]
In Portuguese it is more correct to say "Nabeel Siddiqui é Professor Auxiliar de mídia digital na Susquehanna University"

**Comment by acrymble** [comment]
@programminghistorian/french-team @programminghistorian/spanish-team can I please get confirmation of the translation in the previous comment?

File: _data/ph_authors.yml [hunk 1]
```diff
@@ -921,14 +921,36 @@
...
+      Nabeel Siddiqui is an Assistant Professor of Digital Media at Susquehanna University.
     es: |
-      Nabeel Siddiqui es doctorando en Estudios Americanos en College of William and Mary, donde también es becario del Digital Equality Lab. Su investigación se centra en las relaciones entre las Humanidades y la Informática, en particular en la historia cultural de los medios, Comunicación y Humanidades Digitales, Ciencias de la Información, Historia de la Ciencia y Tecnología. Actualmente, está realizando su tesis doctoral titulada "Byting Out the Public", que pretende estudiar la relación entre el ordenador personal y la privacidad.
+      Nabeel Siddiqui es profesor asistente de medios digitales en la Universidad de Susquehanna.
+    fr: |
+      Nabeel Siddiqui est professeur adjoint de médias numériques à l'Université Susquehanna.
```
Comment (mariechristineb) [inline _data/ph_authors.yml:947]: à l'Université **de** Susquehanna

**Comment by acrymble** [comment]
Thank you @mariechristineb !

**Review by acrymble — APPROVED** [review]

**Pull request merged**